app/pages/learn/01_tutorial/01_your-first-java-app/02_building-with-intellij-idea.md

```diff
@@ -0,0 +1,207 @@
+
+To simplify the view on your project, for example to focus on the package structure, click the title bar to select the desired view from the list.
+
+[![Project view](/assets/images/intellij-idea/project-view.png)](/assets/images/intellij-idea/project-view.png)
+
+Let’s add some code. We’ll start by creating a new class. In the **Project** tool window on the left, select the directory `src/main/java`. Add a new Java file using the shortcut **⌘N** (on macOS) or **Alt+Insert** (on Windows/Linux). Alternatively, right-click the **Project** tool window to open the context menu and select **New | Java class**. Name this class `HelloWorld`.
+
+[![New Java class](/assets/images/intellij-idea/new-java-class.png)](/assets/images/intellij-idea/new-java-class.png)
+
+## Writing and editing code
+Inside the class `HelloWorld`, we will write the main method, which is where execution of our Java program will start.
+The classic way to write a main method in Java is:
+```java
+public static void main(String[] args) {
+  
+}
+```
+As you start typing, you’ll notice that IntelliJ IDEA gives you [code completion](https://www.jetbrains.com/help/idea/auto-completing-code.html). It will help you complete the names of classes, methods, fields, and keywords, and other types of completion.
+
+[![Code completion](/assets/images/intellij-idea/code-completion1.png)](/assets/images/intellij-idea/code-completion1.png)
+
+[![Code completion](/assets/images/intellij-idea/code-completion2.png)](/assets/images/intellij-idea/code-completion2.png)
+
+IntelliJ IDEA will show you if you’ve typed or selected something that doesn’t compile, or if it sees any other problems. If you press **Alt+Enter** it will offer options to fix the problem. You can use **F2** to move to the next problem, and **Shift+F2** to go to the previous problem. IntelliJ IDEA will help you make sure that your syntax is correct and your code can be compiled, by offering suggestions that are context-sensitive.
+
+[![Suggestions](/assets/images/intellij-idea/alt-enter.png)](/assets/images/intellij-idea/alt-enter.png)
+
+To speed up development, we can also [generate code](https://www.jetbrains.com/help/idea/generating-code.html). IntelliJ IDEA can generate constructors, getters and setters, toString(), equals() and hashCode() methods, and more.
+
+[![Generate code](/assets/images/intellij-idea/generate-code.png)](/assets/images/intellij-idea/generate-code.png)
+
+Instead of writing the main method ourselves, we can use [live templates](https://www.jetbrains.com/help/idea/using-live-templates.html) to write it for us. When we write `main`, IntelliJ IDEA will suggest the main live template. Press **Enter** to select the suggestion, and the main method will appear.
```

Comment:
Hey @mlvandijk ,

Live templates are a great feature, but for a person that just wants to write and run some correct Java code, might be too much at this early stage. Can you please remove this reference?

Thank you 😄